src/wristpy/core/cli.py

```diff
@@ -0,0 +1,92 @@
+"""CLI for wristpy."""
+
+import argparse
+import pathlib
+from typing import List, Optional
+
+from wristpy.core import config, orchestrator
+
+logger = config.get_logger()
+settings = config.Settings()
+
+
+def parse_arguments(args: Optional[List[str]] = None) -> argparse.Namespace:
+    """Argument parser for python cli.
+
+    Args:
+        args: Optional argument for when accessed via python script.
+
+    Returns:
+        Namespace object with all of the input arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Run the main wristpy pipeline",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument("input", type=pathlib.Path, help="Path to the input data.")
+
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=pathlib.Path,
+        help="Optional path where data will be saved.",
+    )
+
+    parser.add_argument(
+        "-c",
+        "--calibrator",
+        type=str,
+        choices=["ggir", "gradient", "none"],
+        help="Pick which calibrator to use. Can be 'ggir' or 'gradient'."
+        "Leave empty or enter 'none' to proceed without calibration.",
+    )
+
+    parser.add_argument(
+        "-t",
+        "--thresholds",
+        type=float,
+        nargs=3,
+        default=[
+            settings.LIGHT_THRESHOLD,
+            settings.MODERATE_THRESHOLD,
+            settings.VIGOROUS_THRESHOLD,
+        ],
+        help="Provide three thresholds for light, moderate, and vigorous activity."
+        "values must be given in ascending order. Defaults to config values.",
+    )
+
+    parser.add_argument(
+        "-e",
+        "--epoch_length",
+        type=int,
+        help="Desired sampling rate in seconds. Leave empty to skip downsampling.",
```

Comment:
The epoch length are integers in seconds, so the smallest It can be is 1 second. My assumption is that the user's sampling rate will always be more frequent than 1 Hz. 

To the more general point I took the name from ggir's use of "epoch1", which corresponds to the 5 second temporal resolution, and why we default to 5 second epochs in the orchestrator. 

@Asanto32 Do these two assumptions seem reasonable to you? 

This actually does bring up a question about the default values for this argument, when using the orchestrator in a script the default value is 5, but here when no argument is given it will default to None. Do you consider this inconsistency potentially confusing or is it minor enough to ignore?

Reply:
Maybe rephrase help to mention that this is the window length for base metrics (enmo, anglez, activity)